components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/pom.xml

```diff
@@ -198,6 +198,15 @@
             <artifactId>slf4j-log4j12</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Do we really need Jackson, this is usually reported on security tools

Reply:
removed Jackson since it was unnecessory